src/families/filecoin/bridge/utils/api.ts

```diff
@@ -0,0 +1,107 @@
+import { log } from "@ledgerhq/logs";
+import { AxiosRequestConfig, AxiosResponse } from "axios";
+import { LedgerAPINotAvailable } from "@ledgerhq/errors";
+
+import {
+  BalanceResponse,
+  BroadcastTransactionRequest,
+  BroadcastTransactionResponse,
+  EstimatedFeesRequest,
+  EstimatedFeesResponse,
+  NetworkStatusResponse,
+  TransactionResponse,
+  TransactionsResponse,
+} from "./types";
+import network from "../../../../network";
+import { blockchainBaseURL } from "../../../../api/Ledger";
+import { getCryptoCurrencyById } from "../../../../currencies";
+import { getEnv } from "../../../../env";
+
+const root = "http://127.0.0.1:8888/blockchain/filecoin";
+
+const getFilecoinURL = (path?: string): string => {
+  if (getEnv("DEVELOPMENT_MODE")) return root + (path ? path : "");
```

Comment:
I would prefer not to introduce a "DEVELOPMENT_MODE" but instead have explicit envs. like a "API_FILECOIN_EXPLORER".

Reply:
Perfect! I used that env var because I did not want to create a new one. But I can change it, no problem. I think your suggestion is better.